### DISCUSSION TITLE
Unable to send iperf data traffic through secured tunnel created between two VLAN interfaces

### DISCUSSION THREAD

**Discussion by hbhokare** [strongswan/strongswan#1287] [Q&A]
Hi,

We have created the secured tunnel between two VLAN tagged ethernet interfaces using swanctl.conf and swanctl command line utility. We have following questions based on our experiments. 

1. Able to send the iperf data traffic through secured tunnel when esp=null proposal is given in swanctl.conf 
2. Unable to send the iperf data traffic through secured tunnel when esp=aes256-sha256-ecp256. We have seen packets are getting dropped at sender. 
3. Able to ping (icmp) traffic through secured tunnel when esp=aes256-sha256-ecp256 and esp= null

Here we can see different results with changing esp proposals-

** iperf data traffic through secured tunnel and below proposal working with VLAN**
esp_proposals = null
esp_proposals = aes256gcm16 ( Process by AESNI)
esp_proposals = aes128gcm16 ( Process by AESNI )

** iperf data traffic through secured tunnel and below proposal not working with VLAN**
esp_proposals = aes128-sha1    ( Processed by QAT offload )
esp_proposals = aes256-sha256-ecp256 ( Processed by QAT offload)

Do we need to do some specific setting in IPSec stack. Can you please help us here?

Thanks,
-Harish

**Comment by Thermi** [comment]
> esp_proposals = aes128-sha1 ( Processed by QAT offload )

Can't exactly tell, try to use wireshark and the save-keys plugin to verify the packets are OK.

> sha256

might be a sha256_128 <> sha256_96 difference, try setting `connections.<conn>.children.<child>.sha256_96` to `yes`

Please provide information as shown in #196
